audio/espeak-ng/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                espeak-ng
+github.setup        espeak-ng espeak-ng 1.50
+version             1.50
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeak NG is a multi-lingual software speech synthesizer
+
+long_description    eSpeak NG is a multi-lingual software speech synthesizer
+
+homepage            https://github.com/espeak-ng/espeak-ng
+distname            ${version}
+checksums           rmd160  bd3dd4e17863bb1aa3a964517d888ed9997f0e3e \
+                    sha256  f0505366d5391f1291a9c5cc0816461719f46eaed84cfbf57fb81b0dc381de21 \
+                    size    13665736
+
+depends_lib         port:pcaudiolib \
+                    port:sonic \
+                    port:rb-ronn-ng \
+                    port:rb-kramdown \
+                    port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd       ./autogen.sh && ./configure --with-extdict-ru --with-extdict-zh --with-extdict-zhy
+build.cmd           make src/espeak-ng src/speak-ng en && make
+build.target
```

Comment:
Overriding build.cmd in this way is unusual. Why is this necessary? Does `build.target-append src/espeak-ng src/speak-ng en docs` not work?

`-j8` should not be hardcoded here. If you absolutely must override build.cmd to contain multiple make command invocations, you can get the correct parallel build value using `-j${build.jobs}`.